contrib/libxslt/libxslt_sapi.h

```diff
@@ -0,0 +1,56 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef CONTRIB_LIBXSLT_LIBXSLT_SAPI_H_
+#define CONTRIB_LIBXSLT_LIBXSLT_SAPI_H_
+
+#include <idn2.h>
+#include <syscall.h>
+
+#include <cstdlib>
+
+#include "libxslt_sapi.sapi.h"  // NOLINT(build/include)
+#include "sandboxed_api/util/fileops.h"
+namespace sapi {
+namespace contrib {
+namespace libxslt {
+class LibXsltSapiSandbox : public LibXSLTSandbox {
+ public:
+  std::unique_ptr<sandbox2::Policy> ModifyPolicy(
+      sandbox2::PolicyBuilder*) override {
+    return sandbox2::PolicyBuilder()
+        .AllowSystemMalloc()
+        .AllowDynamicStartup()
+        .AllowOpen()
+        .AllowRead()
+        .AllowStat()
+        .AllowWrite()
+        .AllowExit()
+        .AllowSyscalls({
+            __NR_futex,
+            __NR_close,
+            __NR_lseek,
+            __NR_getpid,
```

Comment:
NIT: Replace __NR_getpid & __NR_gettid with AllowGetPID() and __NR_gettimeofday with AllowTime()